leda/components/Collapsible/index.tsx

```diff
@@ -20,6 +20,8 @@ export const Collapsible = React.forwardRef((props: CollapsibleProps, ref: React
     transition = DEFAULT_TRANSITION,
   } = useProps(props);
 
+  // todo move to a useCallback
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is a bug because you use that function as a dependency

Reply:
Not really get it
Could you explain it?